app/models/friendship.rb

```diff
@@ -4,5 +4,5 @@ class Friendship < ApplicationRecord
 
   scope :is_friend, -> { where('acceptance =?', true) }
   scope :pendant_add, -> { where('acceptance =?', false) }
-  scope :my_friends, ->(id) { where('user_id =? or friend_id =?', id, id) }
+  scope :my_friends, ->(id) { where('friend_id =?', id) }
```

Comment:
Replace unused scope definitions with the associations with SQL queries provided for pending_friendship, inverted friendship, confirmed friendship.